gmail/endpoint.bal

```diff
@@ -22,6 +22,7 @@ import ballerina/http;
 @display {label: "Gmail", iconPath: "GmailLogo.png"}
 public client class Client {
     http:Client gmailClient;
+    string userEmailId = ME;
```

Comment:
Add a comment sating that this denotes the authenticated user.

Reply:
Ack